intranet/apps/eighth/models.py

```diff
@@ -86,13 +86,15 @@ class Meta:
 
     @property
     def name(self):
+        """If show_full_name is set, returns last name, first name. Otherwise, returns last name only."""
         if self.show_full_name and self.first_name:
             return self.last_name + ", " + self.first_name
         else:
             return self.last_name
 
     @property
     def to_be_assigned(self):
+        """ Returns True if the sponsor's name contains "to be assigned" or similar"""
```

Comment:
Needs a period at the end and return type.
```suggestion
        """bool: Returns True if the sponsor's name contains "to be assigned" or similar."""
```